src/seedu/addressbook/common/Utils.java

```diff
@@ -34,4 +35,24 @@ public static boolean elementsAreUnique(Collection<?> items) {
         }
         return true;
     }
+
+    /**
+     * This method converts the set of keywords to a set of lower case strings and returns it.
+     * @return set of lower case string
+     */
+    public static Set<String> keywordToLowerCaseSet(Set<String> keywords) {
```

Comment:
Since you put this method in Utils class, you can generalize this method so that it doesn't specifically mention `keywords`. May be `words` is a more general term. Also, the method name doesn't follow our coding standard.